third_party/binary/BUILD

```diff
@@ -1,16 +1,10 @@
-GO_CI_LINT_VERSION = "1.60.3"
+GO_CI_LINT_VERSION = "1.61.0"
 
 remote_file(
     name = "golangci-lint",
     binary = True,
     exported_files = ["golangci-lint-%s-${OS}-${ARCH}/golangci-lint" % GO_CI_LINT_VERSION],
     extract = True,
-    hashes = [
```

Comment:
I'm not sure we really need the hashes on this, we don't actually build it into anything